legacy/src/app/partials/cards/tags.html

```diff
@@ -5,7 +5,7 @@ <h2 class="p-heading--four">Tags</h2>
   <p>
     <span data-ng-if="!node.tags.length"><i>No tags</i></span>
     <span data-ng-repeat="tag in node.tags">
-      <a href="{$ newURLBase $}/machines?q=tags:({$ tag $})">{$ tag $}</a>
+      <a href="{$ newURLBase $}/controllers?q=tags:({$ tag $})">{$ getTagName(tag) $}</a>
```

Comment:
huh, I wonder how long that hasn't worked?!

Reply:
Probably since the beginning of time